workloads/logging/deploy_logging_stack.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Initialize kubeconfig to get access to the OpenShift cluster
+if [[ -n $KUBECONFIG_PATH ]]; then
+  export KUBECONFIG=$KUBECONFIG_PATH
+else
+  echo "Looks like kubeconfig path is not set, assuming that it's in the default location - $HOME/.kube/config"
```

Comment:
s/echo/log function for consistency with other workloads?
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/3bd15e89dc63fdd545564ddef9bd2abcc51b8dfe/workloads/network-perf/common.sh#L3-L5
I'll create a PR for scale workloads too

Reply:
also print out the cluster version at the start and uuid at the end?
^^ has helped with jenkins earlier to keep track

Reply:
Fixed it, thanks.